publication-event-emitter/src/test/java/no/unit/nva/doi/lambda/DynamodbEventFanoutStreamRecordsEventBridgeHandlerTest.java

```diff
@@ -24,11 +30,22 @@ public void handleRequestWritesToConsoleOnValidEvent() throws Exception {
                 e.printStackTrace();
             }
         };
-        FanoutHandler handler = new FanoutHandler(eventPublisher);
+
+        DynamodbEventFanoutStreamRecordsEventBridgeHandler handler = createHandler(eventPublisher);
         Context context = Mockito.mock(Context.class);
         File eventFile = new File(DYNAMODB_STREAM_EVENT);
         DynamodbEvent event = objectMapper.readValue(eventFile, DynamodbEvent.class);
 
         handler.handleRequest(event, context);
     }
+
+    @BeforeEach
+    void setUp() {
+        environmentMock = mock(Environment.class);
+        when(environmentMock.readEnv(ALLOWED_ORIGIN_ENV)).thenReturn(WILDCARD);
```

Comment:
Why do we need this? Do we expose handler in API Gateway?

Reply:
Fixed.